gulpfile.js

```diff
@@ -0,0 +1,57 @@
+( () => {
+	'use strict';
+
+	const cleanCSS = require( 'gulp-clean-css' ),
+		del = require( 'del' ),
+		env = process.env.NODE_ENV || 'prod',
+		gulp = require( 'gulp' ),
+		rename = require( 'gulp-rename' ),
+		runSequence = require( 'run-sequence' ),
+		sass = require( 'gulp-sass' ),
+		sourcemaps = require( 'gulp-sourcemaps' );
+
+	const paths = {
+		css: [ 'assets/css/*.scss' ]
+	};
+
+	gulp.task( 'clean', function () {
+		return del( [ 'build' ] );
+	} );
+
+	gulp.task( 'css', function () {
+		return gulp.src( paths.css )
+			.pipe( sourcemaps.init() )
+			.pipe( sass( {
+				errLogToConsole: true,
+				outputStyle: 'expanded',
+			} ).on( 'error', sass.logError ) )
+			.pipe( cleanCSS() )
+			.pipe( sourcemaps.write() )
+			.pipe( rename( { suffix: '.min' } ) )
+			.pipe( gulp.dest( 'build/assets/css' ) )
+	} );
+
+	gulp.task( 'php', function() {
+		return gulp.src( '**/*.php' )
```

Comment:
This includes the PHP files within `build`. So when this is run without a `clean` (e.g. during a `gulp watch`) the PHP files within the `build` directory are copied into a `build/build` directory. This happens recursively each time it is triggered by a `watch`.

Reply:
Fixed by https://github.com/woocommerce/sensei-course-progress/pull/42/commits/938a7639bbeb6de156fd3df50cbbb0c0c448e84d.